### ISSUE TITLE
Add relation label to gene product field

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/simple-annoton-editor#75]
Hi @tmushayahama 

I am working on Noctua documentation and in doing so noticed that we don't have a relation for the gene product and MF in the Default form.

Would it be possible to add (MF enabled by) to the right of Gene Product in the Gene Product field?

I think that would complete the relations for all entities and ontology terms in the form.

Thx.

**Comment by tmushayahama** [comment]
@vanaukenk yes, this one can be safely added as Form 2.0 will not create macromolecular complex on the fly. I think has part terms was the issue
